tests/unit/test_metrics.py

```diff
@@ -433,9 +436,84 @@ def test_sum_pairwise_distances_large_block_size(self) -> None:
         # Check output matches expected
         self.assertAlmostEqual(output, expected_output, places=5)
 
-    def test_mmd_block_ints(self) -> None:
+    def test_sum_pairwise_distances_zero_block_size(self):
+        """
+        Test sum_pairwise_distances when a zero block size is given.
+        """
+        # Setup some data
+        x = jnp.array([[0, 0], [1, 1], [2, 2]])
+        y = jnp.array([[0, 0], [1, 1]])
```

Comment:
Can't we just use the existing `self.x` and `self.y`? (Same comment on the rest of the added tests)

Reply:
Updated